docs/product/components/buttons.html

```diff
@@ -418,14 +418,41 @@
                         {% endif %}
                     </td>
                     <td class="va-middle">{{ size.fs }}</td>
-                    <td class="va-middle"><button class="s-btn s-btn__primary ws-nowrap {{ size.class }}" type="button" aria-pressed="false">Ask question</button></td>
+                    <td class="va-middle"><button class="s-btn s-btn__primary ws-nowrap {{ size.class }}" type="button">Ask question</button></td>
                 </tr>
                 {% endfor %}
             </tbody>
         </table>
     </div>
 </section>
 
+<section class="stacks-section">
+    {% header "h2", "Toggle buttons" %}
+    <p class="stacks-copy">Each button class has a selected state which can be visially activated by applying the <code class="stacks-code">.is-selected</code> class. When a button can switch between selected and unselected states, it is important to also annotate the button with the <code class="stacks-code">aria-pressed</code> attribute for accessibility. A <code class="stacks-code">title</code> attribute may also be appropriate to describe what will happen when pressing the button.</p>
+
+    <div class="stacks-preview">
+{% highlight html %}
+<button class="s-btn" type="button" aria-pressed="false" title="…">…</button>
+<button class="s-btn is-selected" type="button" aria-pressed="true" title="…">…</button>
+{% endhighlight %}
+{% highlight javascript %}
+toggleButton.addEventListener('click', () => {
+    let wasSelected = toggleButton.getAttribute('aria-pressed') === 'true';
+    let isSelected = !wasSelected;
+    toggleButton.classList.toggle('is-selected', isSelected);
+    toggleButton.setAttribute('aria-pressed', isSelected.toString());
+    …
+});
```

Comment:
```suggestion

<script>
    toggleButton.addEventListener('click', () => {
        let wasSelected = toggleButton.getAttribute('aria-pressed') === 'true';
        let isSelected = !wasSelected;
        toggleButton.classList.toggle('is-selected', isSelected);
        toggleButton.setAttribute('aria-pressed', isSelected.toString());
        …
    });
</script>
```

This docs presentation was a little wack, so let's just wrap this in script tags within the HTML block.